db/seeds.rb

```diff
@@ -5,4 +5,79 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
-User.create(user_name: "sasaki", email: "a@g.com", password: "aaaaaa")
\ No newline at end of file
+
+label1 = Label.create!(name: "sample1")
+label2 = Label.create!(name: "sample2")
+label3 = Label.create!(name: "sample3")
+label4 = Label.create!(name: "sample4")
+label5 = Label.create!(name: "sample5")
+label6 = Label.create!(name: "sample6")
+label7 = Label.create!(name: "sample7")
+label8 = Label.create!(name: "sample8")
+label9 = Label.create!(name: "sample9")
+label10 = Label.create!(name: "sample10")
+
+@user1 = User.create(user_name: "sasaki", email: "a@g.com", password: "aaaaaa", admin: "true")
```

Comment:
存在しないことが保証されているドメインである`example.com`を使うようにしてください。
```suggestion
@user1 = User.create(user_name: "sasaki", email: "a@example.com", password: "aaaaaa", admin: "true")
```

Reply:
seeds.rbのアドレスのドメインをexample.comに修正しました。